### ISSUE TITLE
You might need to cover other interface types

### ISSUE THREAD

**Issue by technologyevangelist1** [routetehpacketz/cisco-ip-trace#21]
You might need to cover other interface types like ten gig, TwoGigabitEthernet and TwentyFiveGigE also..When mac was larning from ten gig, I felt that code stopped there...I can be wring also..But, just an FYI

**Comment by technologyevangelist1** [comment]
Other thing I see, if there is a phone in the cdp neighbor, it thinks that it is another device and trying ssh and failing..In my case, it was a polycom phone that runs cdp...

**Comment by routetehpacketz** [comment]
Hey @technologyevangelist1,

Regarding your last comment, are you using the latest version of the script? What you are describing was intended to be handled in issue #12, which was merged in pull request #15.

You are correct regarding the other interface types needing to be accounted for, so this issue will be used to document that enhancement.

**Comment by technologyevangelist1** [comment]
I agree with you on the phones. I was not patient enough to check your latest code. Sorry for that. One more thing is, if you could also include the switch IP address(in reality, not all the switches have a DNS entry) it would be awesome. Thanks again for all the work you are doing.

**Comment by routetehpacketz** [comment]
hey @technologyevangelist1 

I've finally gotten around to addressing these open issues. If I add some updates to account for TwoGigabit, FiveGigabit, and FortyGigabit, would you be able to do some testing for me?
